src/core/components/forms/checkboxGroup/checkboxGroup.stories.tsx

```diff
@@ -57,4 +58,38 @@ export const CheckboxCardVariant: Story = {
     ),
 };
 
+/**
+ * Usage of the CheckboxGroup component with CheckboxCard variant and additional content when checked.
+ */
+export const CheckboxCardVariantWithChildren: Story = {
+    render: (props) => (
+        <CheckboxGroup {...props}>
+            <CheckboxCard
+                label="First label"
+                description="First description"
+                avatar="first"
+                tag={{ label: 'First', variant: 'info' }}
+            >
```

Comment:
I'd avoid having multiple stories for the same feature, I'd only keep this on the `RadioCard` / `CheckboxCard` component is enough